app/Filament/Resources/EmulatorResource/RelationManagers/EmulatorReleasesRelationManager.php

```diff
@@ -57,10 +64,11 @@ public function table(Table $table): Table
 
             ])
             ->headerActions([
-
+                Tables\Actions\CreateAction::make(),
```

Comment:
Edge case: Deleted a row and tried to recreate it and got a duplicate key error.

Reply:
This is a tough one - it's happening due to the records being soft deleted.

The only resolution I know if is to allow privileged users to restore these records. Added in latest:

<img width="412" alt="Screenshot 2024-09-25 at 10 00 14 PM" src="https://github.com/user-attachments/assets/06d97aee-0729-451d-8544-cabaf3d100a1">

Reply:
It's not something I would expect to occur in normal use. If it's easier to ignore the error, I'm fine with that.